src/components/settings/PushNotifications/PushNotificationsBanner/index.tsx

```diff
@@ -121,7 +121,12 @@ export const PushNotificationsBanner = ({ children }: { children: ReactElement }
   const isSafeAdded = !!addedSafesOnChain?.[safeAddress]
   const isSafeRegistered = getPreferences(safe.chainId, safeAddress)
   const shouldShowBanner = useDebounce(
```

Comment:
just curious - why do we have to debounce here?

Reply:
I think the issue was that there were too many popups opening up at the same time which looked messy. This way there is a small delay before this popup opens. Not ideal but a bit better on the UX side.